plugins/quetz_conda_suggest/quetz_conda_suggest/api.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from fastapi import APIRouter, Depends, HTTPException, status
+from fastapi.responses import FileResponse
+from sqlalchemy.orm.session import Session
+
+from quetz.deps import get_db
+
+router = APIRouter()
+
+
+@router.get("/api/channels/{channel_name}/{subdir}/conda-suggest")
+def get_conda_suggest(channel_name, subdir, db: Session = Depends(get_db)):
+    map_filename = "{0}.{1}.map".format(channel_name, subdir)
+    map_filepath = os.path.join(
+        os.getcwd(), "channels", channel_name, subdir, map_filename
+    )
+
+    if os.path.isfile(map_filepath):
+        return FileResponse(
+            map_filepath,
+            media_type="application/octet-stream",
+            filename=map_filename,
```

Comment:
what if map_filepath does not exist, the handler will return None

Reply:
Then we should get `404` since we check if it exists or not using `os.path.isfile` and we do get `200` as status code. I think we are just unable to get the output from the `response` object.

Reply:
you're right